app/facade/partner_facade.rb

```diff
@@ -0,0 +1,37 @@
+class PartnerFacade
+
+  attr_reader :partner
+
+  def build_partner(partner_params=nil)
+    @partner = Partner.new(permitted_params(partner_params))
+  end
+
+  def load_partner(id)
+    @partner = Partner.find(id)
+  end
+
+  def update_partner(id, partner_params)
+    self.load_partner(id)
+    @partner.attributes = permitted_params(partner_params)
```

Comment:
That was my initial idea, to move the `save` inside the facade. But we don't have access to the `flash`, `redirect_to` and `render`. 
After discussing this in the scrum we agreed that `save` belongs in the controller.